tracer/src/Datadog.Trace/Configuration/ImmutableTracerSettings.cs

```diff
@@ -463,6 +464,12 @@ internal ImmutableTracerSettings(TracerSettings settings, bool unusedParamNotToU
         /// </summary>
         internal string[] PropagationStyleExtract { get; }
 
+        /// <summary>
+        /// Gets a value indicating whether the propagation should only try
+        /// extract the first header.
+        /// </summary>
+        internal bool PropagationExtractFirst { get; }
```

Comment:
tiny nit: it's a bit verbose, but I would probably have called this `PropagationExtractFirstOnly` to make the behaviour a little clearer

Reply:
I was think the same thing and commented elsewhere before I saw your comment here 😅

Reply:
😬 I've started a slack conversation to see if we can change the environment variable name. I do agree that's better so we could at least change the C# name now and we could update the environment variable name separately if it gets updated

Reply:
Welp it's too late for the environment variable change it seems. But renaming the C# property would be a welcome improvement!